lightningd/pay.c

```diff
@@ -814,6 +814,82 @@ send_payment(struct lightningd *ld,
 	return NULL;
 }
 
+static struct command_result *
+param_route_hop(struct command *cmd, const char *name, const char *buffer,
+		const jsmntok_t *tok, struct route_hop **hop)
+{
```

Comment:
Need to mark UNNEEDED here, and eliminate in next patch.  Otherwise compile bisection is broken :(